openapi.yaml

```diff
@@ -0,0 +1,189 @@
+      operationId: GenerateObjects
+      parameters:
+        - name: schema
+          in: query
+          schema:
+            type: string
+        - name: count
+          in: query
+          schema:
+            type: integer
+            format: int32
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+  /data/values:
+    get:
+      tags:
+        - Data
+      summary: Generate Values
+      operationId: GenerateValues
+      parameters:
+        - name: formula
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: count
+          in: query
+          required: true
+          schema:
+            type: integer
+            format: int32
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: number
+  /schema:
+    get:
+      tags:
+        - Schema
+      summary: Read All Schemas
+      operationId: ReadAllSchemas
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+    post:
+      tags:
+        - Schema
+      summary: Create Schema
+      operationId: CreateSchema
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+              properties:
+                slug:
+                  type: string
+                  description: Unique identifier for the schema
+                fields:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      name:
+                        type: string
+                        description: Name of the field
+                      formula:
+                        type: string
+                        description: Formula for generating data for the field
+              required:
+                - slug
+                - fields
+      responses:
+        '204':
+          description: Schema created successfully
+        '400':
+          description: Bad request
+  /schema/{id}:
+    get:
+      tags:
+        - Schema
+      summary: Read Schema by ID
+      operationId: ReadSchemaById
+      parameters:
+        - name: id
+          in: path
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+  /schema/{id}/delete:
```

Comment:
isn't it just /schema/{id} but with a DELETE method?

Reply:
yes - oops